### DISCUSSION TITLE
How to get the global climate data

### DISCUSSION THREAD

**Discussion by Chengwenju** [iiasa/CWatM#43] [Q&A] [answered]
Hi, everyone
I'm a new user for CWatM, I 'm learning the exercise in tutorial now, but the global climate data in global folder is Nan. the email address provided in documentation is invalid, so, I want to know where I can get the climate dataset?

**Comment by mikhailsmilovic** [comment, marked as answer]
For climate data, access our FTP server by pasting ftp://rcwatm:Water1090@ftp.iiasa.ac.at/ into the address bar of any folder (or however you are comfortable accessing FTP): 
![image](https://user-images.githubusercontent.com/20219171/222757423-bd183d56-92e3-4da2-9ba0-9c5f77d0e7c4.png)

The climate data is GSWP3-W5E5.tar
GSWP3-W5E5_2011_2019.zip is a smaller subset of this data for 2011-2019

**Comment by zzzdbk7** [comment]
i watch your a1course in youtube .i noticed the pathmeteo need P:\ . I can use the fpt ,but i wonder how can i create the p drive with this fpt？
